README.md

```diff
@@ -167,5 +225,15 @@ Write code to do the following:
 
 Write your code here:
 ```ruby
-# code here
+X = Lamp.create(wishes_remaining: 3)
+
+Y = Genie.create(name:"Genie")
+lamp.update(genie: Y)
+
+lamp.update(wishes_remaining: 1)
+Z = Genie.create(name: "Jafar")
+newlamp = Lamp.create(wishes_remaining: 3)
+newlamp.update(genie: Z)
+genie.update(lamp: nil)
+
 ```
```

Comment:
You reference a `lamp` variable but I don't see it defined anywhere. Do you mean `X` instead? Same goes for `genie` and `Y`.

Also, I would get out of the habit of using capitalized variable names (unless convention calls for it, like Ruby class names).